modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/containers/setup-forms/InviteTeamMembersForm/index.js

```diff
@@ -285,93 +311,128 @@ const InviteTeamMembersPage = ({
-							}.com`}
-							selectOnChange={(roleId) =>
-								setFieldValue(
-									`invites[${index}].role`,
-									accountRoles?.find(({id}) => id === +roleId)
-								)
-							}
-						/>
-					))}
-				</ClayForm.Group>
-
-				<div className="mb-4 ml-3 mt-5">
-					<Button
-						className="mr-3 py-2 text-brandy-secondary"
-						displayType="secondary"
-						prependIcon="hr"
-						small
-					>
-						Remove this Member
-					</Button>
-
-					{values?.invites?.length < MAXIMUM_INVITES_COUNT && (
-						<Button
-							className="btn-outline-primary cp-btn-add-members py-2 rounded-xs"
-							onClick={() => {
-								setBaseButtonDisabled(false);
-								setFieldValue('invites', [
-									...values?.invites,
-									getInitialInvite(accountMemberRole),
-								]);
-							}}
-							prependIcon="plus"
-							small
-						>
-							Add More Members
-						</Button>
-					)}
-				</div>
-			</div>
-
-			<div className="invites-helper px-3">
-				<div className="mx-3 pt-3">
-					<h5 className="text-neutral-7">
-						{`${
-							projectHasSLAGoldPlatinum
-								? ROLE_TYPES.requestor.name
-								: ROLE_TYPES.admin.name
-						}	roles available: ${availableAdminsRoles} of ${maxRequestors}`}
-					</h5>
-
-					<p className="mb-0 text-neutral-7 text-paragraph-sm">
-						{`Only ${maxRequestors} member${
-							maxRequestors > 1 ? 's' : ''
-						} per project (including yourself) have
-						 role permissions (Admins & Requestors) to open Support
-						 tickets. `}
-
-						<a
-							className="font-weight-bold text-neutral-9"
-							href={supportLink}
-							rel="noreferrer"
-							target="_blank"
-						>
-							Learn more about Customer Portal roles
-						</a>
-					</p>
-				</div>
-			</div>
+			<FieldArray
+				name="invites"
+				render={({pop, push}) => (
+					<>
+						<div className="invites-form overflow-auto px-3">
+							<div className="px-3">
+								<label>Project Name</label>
+
+								<p className="invites-project-name text-neutral-6 text-paragraph-lg">
+									<strong>{project.name}</strong>
+								</p>
+							</div>
+
+							<ClayForm.Group className="m-0">
+								{values?.invites?.map((invite, index) => (
+									<TeamMemberInputs
+										disableError={hasInitialError}
+										id={index}
+										invite={invite}
+										key={index}
+										options={accountRolesOptions}
+										placeholderEmail={`username@${
+											project?.code?.toLowerCase() ||
+											'example'
+										}.com`}
+										selectOnChange={(roleId) =>
+											setFieldValue(
+												`invites[${index}].role`,
+												accountRoles?.find(
+													({id}) => id === +roleId
+												)
+											)
+										}
+									/>
+								))}
+							</ClayForm.Group>
+
+							{showEmptyEmailError && (
+								<Badge>
+									<span className="pl-1">
+										Please enter your email address.
+									</span>
+								</Badge>
+							)}
```

Comment:
We need to use our approach to show error, integrate it with <Input> and formik. Maybe use a setError with setTouched.

Let's try avoid to use errors outside the Input
```suggestion
```